Jenkinsfile

```diff
@@ -47,27 +50,26 @@ pipeline {
     Updating generated files for Beat.
     Checks the GO environment.
     Checks the Python environment.
-    Checks YAML files are generated. 
+    Checks YAML files are generated.
     Validate that all updates were committed.
     */
-    stage('Intake') { 
+    stage('Intake') {
       agent { label 'linux && immutable' }
       options { skipDefaultCheckout() }
       environment {
         PATH = "${env.PATH}:${env.WORKSPACE}/bin"
         HOME = "${env.WORKSPACE}"
         GOPATH = "${env.WORKSPACE}"
       }
-      when { 
+      when {
         beforeAgent true
         expression { return params.intake_ci }
       }
       steps {
-        withEnvWrapper() {
```

Comment:
Why is this removed?

Reply:
this step was created as a workaround until we can use credentials for environment variables like NOTIFY_TO, now we have the credentials created so it is no longer necessary.